tests/auxiliary/minicore.rs

```diff
@@ -41,12 +41,24 @@ macro_rules! impl_marker_trait {
 }
 
 #[lang = "pointee_sized"]
+#[diagnostic::on_unimplemented(
+    message = "values of type `{Self}` may or may not have a size",
+    label = "may or may not have a known size"
+)]
```

Comment:
Suggestion: if we decide to go with this, can you make sure to leave a comment mentioning needing to sync the diagnostics attribute in the real `core`?

Reply:
I added a note at the top of `minicore.rs`, or did you mean to add the note in  `library/core`?